### DISCUSSION TITLE
Provide a way to empty the trash

### DISCUSSION THREAD

**Discussion by rodrcastro** [GitbookIO/community#25] [Feature Requests]
### What's your feature/idea?

I want to be able to empty the trash right away if I choose.

### What's your desired outcome?

The idea here is being able to get rid of everything inside the trash with a click of a button without having to wait the 7 days for deletion.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

Having the 7 days is nice in case we want to recover something so it's still good to have that. I'd just like to have the option to empty the trash right away if needed.

**Comment by emmerich** [comment]
@rodrcastro Is this coming from a data security / content privacy use-case?

**Reply by rodrcastro** [reply]
That's exactly right @emmerich.
